mkdocs_monorepo_plugin/parser.py

```diff
@@ -138,6 +137,44 @@ def read(self):
         try:
             with open(self.absNavPath, 'rb') as f:
                 self.navYaml = yaml_load(f)
+
+                # This will check if there is a `docs_dir` property on the `mkdocs.yml` file of
+                # the sub folder and scaffold the `nav` property from it
+                if self.navYaml and 'nav' not in self.navYaml:
+                    if "docs_dir" in self.navYaml:
+                        docsDirPath = os.path.join(os.path.dirname(self.absNavPath), self.navYaml["docs_dir"])
+
+                        def navFromDir(path):
+                            directory = {}
+
+                            for dirname, dirnames, filenames in os.walk(path):
+
+                                if dirname == docsDirPath:
+                                    dn = os.path.basename(dirname)
+                                else:
+                                    dn = dirname_to_title(os.path.basename(dirname))
+                                directory[dn] = []
+
+                                for dirItem in dirnames:
+                                    subNav = navFromDir(path=os.path.join(path, dirItem))
+                                    if subNav:
+                                        directory[dn].append(subNav)
+
+                                for fileItem in filenames:
+                                    fileName, fileExt = os.path.splitext(fileItem)
+                                    if fileExt == '.md':
+                                        fileTitle = get_markdown_title(fileName)
+                                        filePath = os.path.join(os.path.relpath(path, docsDirPath), fileItem)
+                                        directory[dn].append({fileTitle: filePath})
+
+                                if len(directory[dn]) == 0 or directory[dn] == [{}]:
+                                    del directory[dn]
+
+                                if directory != {}:
+                                    return directory
+
+                        self.navYaml["nav"] = navFromDir(docsDirPath)[os.path.basename(docsDirPath)]
+
```

Comment:
Hey! Appreciate you working on this feature.

If you could add some tests for this https://github.com/backstage/mkdocs-monorepo-plugin/tree/master/__tests__, it will really help out in reviewing, thanks!

Most of it looks good to me! :)